src/components/panels/NewsPanel.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <HubPanelWrap :title="headertext">
+    <b-carousel id="infoscreen-carousel" :interval="9000" :indicators="false" :controls="true">
+      <b-carousel-slide
+        v-for="(newsitem,index) in news"
+        :key="index"
+        v-if="newsitem.translations['en']"
+        :text="title(newsitem)"
+        v-on:click.native="open(newsitem.link)"
+      >
+        <span slot="img" class="middling-helper"></span>
+        <b-img slot="img" class="d-block slide-image" fluid center :src="getimgling(newsitem.imageLink)" :alt="title(newsitem)" />
+      </b-carousel-slide>
+    </b-carousel>
+  </HubPanelWrap>
+</template>
+<style>
+.carousel-caption {
+  background-color: rgba(64, 64, 64, 0.7) !important;
+  color: white;
+  width: 100% !important;
+  left: 0 !important;
+  bottom: 0 !important;
+  font-weight: bolder;
+  padding-bottom: 0px !important;
+}
+.carousel-caption a {
+  color: white;
+}
+.carousel-item {
+  max-height: 250px;
+  min-height: 230px;
+}
+
+.middling-helper {
+  display: inline-block;
+  vertical-align: middle;
+  height: 100%;
+}
+</style>
+<script>
+import HubPanelWrap from '@/components/HubPanelWrap';
+
+export default {
+  props: ['news'],
+  name: 'NewsPanel',
+  components: { HubPanelWrap },
+  computed: {
+    headertext() {
+      return 'News';
+    }
+  },
+  methods: {
+    open: function(url) {
+      window.open(url, '_blank');
+    },
+    getimgling: function(url) {
+      return 'https://img.mybitti.de/o_webp/' + url;
```

Comment:
Do we need this from a third party?

Reply:
at the moment, yes, it cleaned up some security issues and we don't really have anywhere to proxy this on our own at the moment. could probably set a proxy of our own at some point, as well, but there's no good way within vue itself to, on the fly, convert a png to a webp without something like this.